build.gradle

```diff
@@ -0,0 +1,20 @@
+group 'com.waverley'
+version '0.0.1'
+
+apply plugin: 'java'
+
+sourceCompatibility = 1.8
+
+repositories {
+    jcenter()
+}
+
+dependencies {
+    compile('org.testng:testng:6.12')
+}
+task wrapper(type: Wrapper) {
+    gradleVersion '4.2.1'
+}
+test {
+    useTestNG() { suites 'src/test/resources/smoke-suite.xml' }
+}
```

Comment:
missing new line at the end of the file... should you add it to all you java and config files...